docs/integrations/sources/intercom.md

```diff
@@ -48,9 +48,10 @@ The Intercom connector should not run into Intercom API limitations under normal
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                                                       |
-|:--------| :--------- | :------------------------------------------------------- | :-------------------------------------------------------------------------------------------- |
-| 0.1.30  | 2023-01-27 | [22010](https://github.com/airbytehq/airbyte/pull/22010) | Set `AvailabilityStrategy` for streams explicitly to `None`                                                     |
-| 0.1.29  | 2022-10-31 | [18681](https://github.com/airbytehq/airbyte/pull/18681) | Define correct version for airbyte-cdk~=0.2                                                    |
+|:--------|:-----------| :------------------------------------------------------- |:----------------------------------------------------------------------------------------------|
+| 0.1.31  | 2023-01-30 | [22095](https://github.com/airbytehq/airbyte/pull/22095) | Extended `Contacts` scehma adding `opted_out_subscription_types` property                     |
```

Comment:
Hint: I just found a typo, should be: schema

Reply:
Ah yes, all set